frontend/src/main.tsx

```diff
@@ -7,6 +7,7 @@ import {
 } from 'react-router-dom';
 import Home from './pages/Home/index.tsx';
 import ListContainers from './pages/ListContainers/index.tsx';
+import LoadingState from'./context/loading/LoadingState.jsx';
```

Comment:
Since we're working with the .tsx extensions we do not need to add it explicitly

```suggestion
import LoadingState from'./context/loading/LoadingState';
```